blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -1,226 +1,265 @@
-					customFields={{
-						adType: "300x250_gallery",
-						displayAdLabel: true,
-					}}
-				/>
+	const getLeadArtContent = (lead_art) => {
+		if (lead_art.type === "raw_html") {
+			return (
+				<Static id={`${BLOCK_CLASS_NAME}-${lead_art._id}`}>
+					<div dangerouslySetInnerHTML={{ __html: lead_art.content }} />
+				</Static>
 			);
-		} catch (e) {
-			/* istanbul ignore next */
-			AdBlock = () => <p>Ad block not found</p>;
 		}
 
-		if (content.promo_items && (content.promo_items.lead_art || content.promo_items.basic)) {
-			const lead_art = content.promo_items.lead_art || content.promo_items.basic;
-			let lightbox = null;
-			let caption = null;
-
-			if (lead_art.type === "raw_html") {
-				return (
-					<div className="lead-art-wrapper">
-						<Static id={lead_art._id}>
-							<div
-								className="inner-content"
-								dangerouslySetInnerHTML={{ __html: lead_art.content }}
-							/>
-						</Static>
-					</div>
-				);
-			}
-
-			if (lead_art.type === "video") {
-				return (
-					<VideoPlayerPresentational
-						id={id}
-						embedMarkup={lead_art?.embed_html}
-						enableAutoplay={!!customFields?.enableAutoplay}
-						shrinkToFit={customFields?.shrinkToFit}
+		if (lead_art.type === "video") {
+			const embedMarkup = formatPowaVideoEmbed(lead_art?.embed_html, {
+				autoplay: customFields?.enableAutoplay,
+				playthrough: customFields?.playthrough,
+			});
+			return (
+				<MediaItem
+					caption={!hideCaption ? lead_art?.description?.basic : null}
+					credit={!hideCredits ? formatCredits(lead_art.credits) : null}
+					title={!hideTitle ? lead_art?.headlines?.basic : null}
+				>
+					<Video
+						aspectRatio="16 / 9"
+						embedMarkup={embedMarkup}
 						viewportPercentage={customFields?.viewportPercentage}
-						customFields={{
-							playthrough: !!customFields?.playthrough,
-						}}
-						displayTitle={!hideTitle}
-						displayCaption={!hideCaption}
-						displayCredits={!hideCredits}
-						subtitle={lead_art?.headlines?.basic}
-						caption={lead_art?.description?.basic}
-						credits={lead_art.credits}
 					/>
-				);
-			}
+				</MediaItem>
+			);
+		}
 
-			if (lead_art.type === "image") {
-				lightbox = (
-					<>
-						{isOpen && (
-							<Lightbox
-								mainSrc={this.lightboxImgHandler()}
-								onCloseRequest={this.setIsOpenToFalse}
-								imageCaption={!hideCaption ? lead_art.caption : null}
-							/>
-						)}
-					</>
-				);
+		if (lead_art.type === "image") {
+			return (
+				<MediaItem
+					caption={!hideCaption ? lead_art.caption : null}
+					credit={!hideCredits ? formatCredits(lead_art.credits) : null}
+					title={!hideTitle ? lead_art.subtitle : null}
+				>
+					<Button
+						iconLeft={<Icon name="Fullscreen" fill="#6B6B6B" />}
+						label={
+							!isOpen
+								? phrases.t("lead-art-block.fullscreen-enter")
+								: phrases.t("lead-art-block.fullscreen-exit")
+						}
+						onClick={toggleFullScreen}
+					>
+						<span>{buttonLabel}</span>
```

Comment:
No need for `<span>` here - The button uses a `<span>` which ends up giving us nested elements

```suggestion
						{buttonLabel}
```

Reply:
Got it, removed the span.